tests/decode/test5_strange/floats.py

```diff
@@ -0,0 +1,24 @@
+import struct
```

Comment:
Should there be a copyright here?

Reply:
Where is this file used? If it was only used once for generating some data I guess it should not be kept in this repo as it is prone for rotting.